src/applications/facility-locator/components/ServicesAtFacility.jsx

```diff
@@ -91,13 +90,28 @@ class ServicesAtFacility extends Component {
       },
     } = this.props;
 
-    if (!services.benefits || isEmpty(services.benefits.standard)) {
+    if (
+      !services.benefits ||
+      services.benefits.length === 0 ||
+      (services.benefits.standard && services.benefits.standard.length === 0)
+    ) {
       return null;
     }
 
+    /**
+     * Since moving to v1 the services object changed. For now,
+     * supporting v0 as the implementation is still on flipper.
+     *
+     * v0  - services.benefits.standard
+     * v1  - services.benefits
+     */
     return (
       <div className="mb2">
-        <ul>{services.benefits.standard.map(s => this.renderService(s))}</ul>
+        <ul>
+          {(services.benefits.standard &&
```

Comment:
Can also write this like: (this is subjective so feel free to disregard if you want lol)

```jsx
<ul>
  {!isEmpty(services?.benefits?.standard) ? (
    services?.benefits?.map(service => this.renderService(service))
  ) : (
    services?.benefits?.standard?.map(service => this.renderService(service))
  )}
</ul>
```

Reply:
`isEmpty` would be coming from `import { isEmpty } from 'lodash';`